scripts/blogView.js

```diff
@@ -0,0 +1,17 @@
+var blogView = {};
+
+blogView.setTeasers = function() {
+  $('.blog-body *:nth-of-type(n+2)').hide();
```

Comment:
The element with a class of blog-body doesn't have any children, and doesn't seem to be attached to anything else, so this doesn't work.